app/views/admins/settings/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= link_to "Edit Settings", edit_admin_settings_path %></h1>
+
+<table border="1">
+  <tr>
+    <th>NPO Status</th>
+    <th>Student Status</th>
+    <th>Settings ID</th>
+    <th>Current Semester ID</th>
+  </tr>
+  <tr>
+    <td><%= @settings.npo_app_open %></td>
+    <td><%= @settings.student_app_open %></td>
+    <td><%= @settings.id %></td>
+    <td><%= Semester.find(@settings.current_semester_id).season_with_year %></td>
```

Comment:
Why don't we write a method on the `Settings` model so that we can just call:

    @settings.current_semester

Which returns the semester with the `current_semester_id`. If you override the `to_s` method of `Semester` then it'll get called automatically for display I think.